grunt/tasks/jest.js

```diff
@@ -1,84 +1,17 @@
 'use strict';
 
-var async = require('async');
 var fs = require('fs');
-var glob = require('glob');
 var grunt = require('grunt');
 var path = require('path');
 
-var rootPath = path.resolve('.');
```

Comment:
All this code path was really weird and not needed anymore. We can just run `jest --coverage` to do the same thing.